packages/mockyeah-cli/README.md

```diff
@@ -1,14 +1,15 @@
 # mockyeah-cli
 
-Command Line Utility for [mockyeah](https://github.com/ryanricard/mockyeah)
+Command line utility for [mockyeah](https://github.com/ryanricard/mockyeah),
+a powerful service mocking, recording, and playback utility.
 
-Installation:
+<img src="docs/book/logo/mockyeah.png" height="200" />
```

Comment:
This path doesn't appear to be correct.

Reply:
It should work once published, it’s the same as the mockyeah package's one.

Reply:
👍